src/loader/BlendLoader.py

```diff
@@ -73,28 +106,88 @@ def run(self):
         else:
             entities = None
 
-        previously_loaded_objects = set(bpy.context.scene.objects)
+        with bpy.data.libraries.load(path) as (blend_file_data, data_to):
+            self.blend_file_version = [lib.version for lib in bpy.data.libraries if lib.filepath == path]
+            data_block_name = load_from.strip("/")
```

Comment:
I think we should just drop this slash altogether (from the param in the config and just give a block name), it was always confusing.

Reply:
dropped

Reply:
Just make sure that corresponding changes are made in other examples using BlendLoader.

Reply:
> dropped 

It is still in the docu at some places?

> Just make sure that corresponding changes are made in other examples using BlendLoader.

And that the old thing is still supported.